src/hero/hero.css

```diff
@@ -0,0 +1,179 @@
+@import "../index.css";
+
```

Comment:
1. Theme and color scheme could have been more consistent
2. Hero or the landing page suggests that colors of night sky, stars, moon and clouds would be used on the inside.
3. This however is not the case as the colors on the inside are various shades of orange, which are good, but different from what was conveyed outside